content/en/docs/reference/glossary/workload.md

```diff
@@ -1,24 +1,23 @@
 ---
-title: Workloads
+title: Workload
 id: workloads
 date: 2019-02-13
 full_link: /docs/concepts/workloads/
 short_description: >
-   Workloads are objects you use to manage and run your containers on the cluster.
+   A workload is an application running on Kubernetes.
 
 aka: 
 tags:
 - fundamental
-- core-object
 - workload
 ---
-  Workloads are objects you use to manage and run your containers on the cluster.
+   A workload is an application running on Kubernetes.
 
 <!--more--> 
 
-Kubernetes performs the 
-deployment and updates the workload with the current state of the application.
-Workloads include the DaemonSet, Deployments, Jobs, Pods, ReplicaSet, ReplicationController, and StatefulSet objects.
+Various core objects that represent different types or parts of a workload
+include the DaemonSet, Deployment, Job, Pod, ReplicaSet, ReplicationController,
```

Comment:
```suggestion
include the DaemonSet, Deployment, Job, Pod, ReplicaSet,
```

How about dropping ReplicationController? The name is a bit confusing (compare “the Deployment controller” vs. “the ReplicationController controller”).